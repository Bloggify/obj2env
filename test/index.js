import test from 'ava';
import envGenerate from '../src';

test('envGenerate is defined', t => {
	t.is(typeof envGenerate, "function");
});

test('envGenerate throws error when object is not passed as argument', t => {

	const error = t.throws(() => {
		envGenerate();
	});
	t.is(error.message, 'Expected a json object as argument');

});

test('envGenerate does not thrown an error when object is passed as argument', t => {

	t.notThrows(() => {
		envGenerate({});
	});

});

test('envGenerate returns an array when passed a valid json object', t => {

	const result = envGenerate({ a : 1});
	t.snapshot(result);
	t.is(result instanceof Array, true);

});
