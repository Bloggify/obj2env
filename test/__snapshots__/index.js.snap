// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`envGenerate returns an array when passed a valid json object 1`] = `
Array [
  "a=1",
]
`;
